.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
```

Comment:
How is this different from running super linter in actions?

Reply:
Removed super linter.